test/unit/lib/plugins/package/lib/packageService.test.js

```diff
@@ -436,65 +81,52 @@ describe('lib/plugins/package/lib/packageService.test.js', () => {
-      // In service wide artifact, confirm ".serverless-plugins/index.js" is not packaged
-      //
-      // Replaces
-      // https://github.com/serverless/serverless/blob/b12d565ea0ad588445fb120e049db157afc7bf37/test/unit/lib/plugins/package/lib/packageService.test.js#L87-L96
+    it('should exclude default plugins localPath', () => {
+      expect(serviceZippedFiles).to.not.include('.serverless-plugins/index.js');
     });
 
-    it.skip('TODO: should support `package.exclude`', () => {
-      // In service wide and functonIndividually artifact, confirm:
-      // - "dir1/subdir1/index.js" is not packaged
-      // - "dir1/subdir3/index.js" is packaged
-      //
-      // Replace
-      // https://github.com/serverless/serverless/blob/b12d565ea0ad588445fb120e049db157afc7bf37/test/unit/lib/plugins/package/lib/packageService.test.js#L128-L145
-      // https://github.com/serverless/serverless/blob/b12d565ea0ad588445fb120e049db157afc7bf37/test/unit/lib/plugins/package/lib/packageService.test.js#L637-L649
+    it('should support `package.exclude`', () => {
+      expect(serviceZippedFiles, fnIndividualZippedFiles).to.not.include('dir1/subdir1/index.js');
+      expect(serviceZippedFiles, fnIndividualZippedFiles).to.include('dir1/subdir3/index.js');
     });
 
-    it.skip('TODO: should support `package.include`', () => {
-      // In service wide  and functonIndividually artifact, confirm:
-      // - "dir1/subdir2/index.js" is packaged
-      // - "dir1/subdir2/subsubdir1/index.js" is not packaged
-      // - "dir1/subdir2/subsubdir2/index.js" is packaged
-      //
-      // Replaces
-      // https://github.com/serverless/serverless/blob/b12d565ea0ad588445fb120e049db157afc7bf37/test/unit/lib/plugins/package/lib/packageService.test.js#L43-L50
-      // https://github.com/serverless/serverless/blob/b12d565ea0ad588445fb120e049db157afc7bf37/test/unit/lib/plugins/package/lib/packageService.test.js#L625-L635
-      // https://github.com/serverless/serverless/blob/b12d565ea0ad588445fb120e049db157afc7bf37/test/unit/lib/plugins/package/lib/packageService.test.js#L651-L662
+    it('should support `package.include`', () => {
+      expect(serviceZippedFiles, fnIndividualZippedFiles).to.include('dir1/subdir2/index.js');
+      expect(serviceZippedFiles, fnIndividualZippedFiles).to.not.include(
+        'dir1/subdir2/subsubdir1/index.js'
+      );
+      expect(serviceZippedFiles, fnIndividualZippedFiles).to.include(
+        'dir1/subdir2/subsubdir2/index.js'
+      );
     });
 
     it.skip('TODO: should support `functions[].package.individually`', () => {
       // Confirm there's "functions.fnIndividual.package.artifact"
       // Replace
       // https://github.com/serverless/serverless/blob/b12d565ea0ad588445fb120e049db157afc7bf37/test/unit/lib/plugins/package/lib/packageService.test.js#L201-L225
+      expect(serverless.service.getFunction('fnIndividual').package.artifact).to.include(
```

Comment:
It looks great - you're properly confirming that the was an individual artifact created for the function :+1:

Reply:
Indeed. Sorry, for some reason I wasn't able to get `package.artifact` from `serverless.service.getFunction('fnIndividual')` - it was a PEBKAC. I struggle with `fnService` though. 
I suspect adding definition with `fnService` to the `function` configuration block is required, e.g.:
```js
 fnService: {
   name: 'fnService',
 },
```

But then I end up with `artifact.zip` as `package.artifact`.